package.json

```diff
@@ -13,14 +13,22 @@
     },
     "scripts": {
         "eslint": "eslint .",
-        "eslint:fix": "eslint --fix ."
+        "eslint:fix": "eslint --fix .",
+        "lint-structured-files": "./structured-files/lint-structured-files.sh --ignore-path .gitignore --check",
+        "lint-structured-files:fix": "./structured-files/lint-structured-files.sh  --ignore-path .gitignore --write"
     },
     "dependencies": {
         "eslint": "^5.16.0",
         "eslint-config-airbnb-base": "^13.1.0",
         "eslint-plugin-import": "^2.16.0",
-        "eslint-plugin-promise": "^4.1.1"
+        "eslint-plugin-promise": "^4.1.1",
+        "prettier": "^1.18.2",
+        "realpath": "^1.1.0"
```

Comment:
Why is the `realpath` dependency needed? The `lint-structured-files.sh` script is bash, where `realpath` is provided by coreutils.

Reply:
`realpath` is not installed by default on Travis currently (old Debianized(?) versions of coreutils did not include it). We could explicitly install it in `.travis.yml` instead. Does macOS have `realpath` by default?